src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -110,10 +110,19 @@ configurationRegistry.registerConfiguration({
 		},
 		'terminal.integrated.shellArgs.windows': {
 			markdownDescription: nls.localize('terminal.integrated.shellArgs.windows', "The command line arguments to use when on the Windows terminal. [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration)."),
-			type: 'array',
-			items: {
-				type: 'string'
-			},
+			'anyOf': [
+				{
+					type: 'array',
+					items: {
+						type: 'string',
+						description: nls.localize('terminal.integrated.shellArgs.windows.array', "An array of string of argments to use when on the Windows terminal. [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration).")
+					},
+				},
+				{
+					type: 'string',
+					description: nls.localize('terminal.integrated.shellArgs.windows.string', "A single string of argments to use when on the Windows terminal. [Read more about configuring the shell](https://code.visualstudio.com/docs/editor/integrated-terminal#_configuration)."),
```

Comment:
We should explain that the string is in "command-line" format, as described in https://msdn.microsoft.com/en-au/08dfcab2-eb6e-49a4-80eb-87d4076c98c6